packages/loopring_v3/contracts/amm/libamm/AmmWithdrawal.sol

```diff
@@ -32,6 +34,30 @@ library AmmWithdrawal
         S.exchange.withdrawFromApprovedWithdrawals(owners, tokens);
     }
 
+    // This is to claim other people depositing into this AMM pool.
```

Comment:
I guess same as with `withdrawFromApprovedWithdrawals`, but with the difference that people shouldn't deposit to the pool directly so should never happen. :)

`withdrawFromDepositRequest` can also be called by anyone for this smart contract, so maybe just make this contract `Drainable` for all tokens except the pool token so all funds owned by this smart contract can be withdrawn? Someone else could in theory call `withdrawFromDepositRequest` before this function is used and funds would be stuck in this contract.